src/plugins/markdown-with-image/markdown-with-image-display.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import classNames from 'classnames';
+import { IMAGE_POSITION } from './constants.js';
+import Markdown from '../../components/markdown.js';
+import ClientConfig from '../../bootstrap/client-config.js';
+import { getAccessibleUrl } from '../../utils/source-utils.js';
+import { useService } from '../../components/container-context.js';
+import CopyrightNotice from '../../components/copyright-notice.js';
+import { sectionDisplayProps } from '../../ui/default-prop-types.js';
+
+export default function MarkdownWithImageDisplay({ content }) {
+  const clientConfig = useService(ClientConfig);
+  const { text, image } = content;
+
+  const imageSrc = getAccessibleUrl({ url: image.sourceUrl, cdnRootUrl: clientConfig.cdnRootUrl });
+  const imageWrapperStyle = { float: image.position };
+  if (image.position === IMAGE_POSITION.left) {
+    imageWrapperStyle.paddingLeft = 0;
+  } else {
+    imageWrapperStyle.paddingRight = 0;
+  }
```

Comment:
It would be nicer to extract these inline styles into two classes:

* `MarkdownWithImageDisplay-imageWrapper--left`
* `MarkdownWithImageDisplay-imageWrapper--right`

Reply:
Makes sense and cleans up the code, done, thanks